aggregator/src/main/scala/weco/concepts/aggregator/ConceptsAggregator.scala

```diff
@@ -27,9 +26,9 @@ class ConceptsAggregator(
     indexName = indexName
   ).flow
 
-  def run: Future[Done] = {
+  def run(jsonSource: Source[String, NotUsed]): Future[Done] = {
```

Comment:
Moved into run to split ConceptsAggregator creation and running and allow it to be created before you know what the JSON is.

It is now created during construction, so that creation can be common between CLI and lambda mode.